plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -66,6 +67,19 @@
   clone:
     description:
       - Name of VM to be cloned. If C(vmid) is setted, C(clone) can take arbitrary value but required for initiating the clone.
+  cicustom:
+    description:
+      - cloud-init: Specify custom files to replace the automatically generated ones at start.
+  cipassword:
+    description:
+      - cloud-init: Password to assign the user. Using this is generally not recommended. Use ssh keys instead. Also note that older cloud-init versions do not support hashed passwords.
```

Comment:
```suggestion
      - "cloud-init: Password to assign the user. Using this is generally not recommended. Use ssh keys instead. Also note that older cloud-init versions do not support hashed passwords."
```
when there's `:` in a sentence, we must quote the sentence.
is `cloud-init` necessary? imo, it would look better without it.